chat/input-bar/src/InputBar/InputBar.spec.tsx

```diff
@@ -302,23 +302,14 @@ describe('packages/input-bar', () => {
   });
 
   describe('status states', () => {
-    test('renders loading state with default message when state is "loading"', () => {
+    test('renders loading state when state is "loading"', () => {
```

Comment:
Should we add a unit test that verifies the prop/context value is consumed here?